autogpts/autogpt/autogpt/command_decorator.py

```diff
@@ -19,52 +13,58 @@
 
 
 def command(
-    name: str,
-    description: str,
-    parameters: dict[str, JSONSchema],
-    enabled: Literal[True] | Callable[[Config], bool] = True,
-    disabled_reason: Optional[str] = None,
-    aliases: list[str] = [],
-    available: bool | Callable[[BaseAgent], bool] = True,
+    names: list[str] = [],
+    description: Optional[str] = None,
+    parameters: dict[str, JSONSchema] = {},
+    is_valid: Callable[[CommandArgs], ValidityResult] = lambda _: ValidityResult(True),
```

Comment:
It's from [this PR](https://github.com/Significant-Gravitas/AutoGPT/pull/7030). It's supposed to check if a command execution with some parameters is valid just before its execution. It replaced duplication check, I don't think it makes sense, can't we just raise inside command like previously? I'm going to remove `is_valid` unless you stop me! 😄 
I merged that PR with this PR because it touched on the same code and without this I'd have to port irrelevant and bugged code.